client/src/main/java/com/kritsit/casetracker/client/domain/services/Administrator.java

```diff
@@ -123,5 +123,19 @@ private InputToModelParseResult validate(Map<String, Object> inputMap){
 
        return result;
     }
+
+    public int resetPassword(Map<String, Object> inputMap) {
```

Comment:
I think it would be simpler to take it as 2 parameters. The reason for the Map was because the data was yet to be validated to fit the model